src/views/Checkout.vue

```diff
@@ -367,5 +371,7 @@ export default class Checkout extends Vue {
 <style>
     #logo-checkout {
         margin-left: 0.75rem;
+        text-transform: none;
+        letter-spacing: initial;
```

Comment:
These rules probably don't hurt, but why are they necessary?

Reply:
`text-transform` and `letter-spacing` are set on the parent with class `logo` to non-initial values.
I moved this to `App.vue` as a general style rule now.